src/components/Button.tsx

```diff
@@ -0,0 +1,16 @@
+interface button {
+  text: string;
+  outline: string;
+  color: string;
+}
+
+const Button = ({ text, outline, color }: button) => {
+
+  return (
+      <button className={`btn btn-${color}-${outline}`}>
```

Comment:
className can then become:
`className={btn btn-${color}-${primary ? 'filled' : 'outline'}`
This will means it'll return a primary (filled) button if the prop primary is set to true. If it's set to false (or omitted completely), it will return an outline button.